bootstrapvz/common/tasks/grub.py

```diff
@@ -61,10 +99,14 @@ def run(cls, info):
 				device_map.write('(hd0) {device_path}\n'.format(device_path=device_path))
 				if not isinstance(p_map, partitionmaps.none.NoPartitions):
 					for idx, partition in enumerate(info.volume.partition_map.partitions):
-						device_map.write('(hd0,{prefix}{idx}) {device_path}\n'
-						                 .format(device_path=partition.device_path,
-						                         prefix=partition_prefix,
-						                         idx=idx + 1))
+						device_map.write(
+							'(hd0,{prefix}{idx}) {device_path}\n'
+							.format(
+								device_path=partition.device_path,
+								prefix=partition_prefix,
+								idx=idx + 1
+							)
+						)
```

Comment:
Sorry if you misunderstood me, Apollo. I never intended to say that you don't know how to program of something like that.

Please take a look at the following screenshots, where I configured my editor to show these invisible characters:

Tabs + spaces:

![][spaces]

Only tabs:

![][tabs]

Can you see the subtle differences? In this case, it was just one char wide (the tab version was shifted to the left), but I guess you can imagine what this can mean in other parts of the code.

What I'd like to make clear here is: we only use tabs if we are in a different block level of the code. The `.format()` method and its arguments, in this example, does not constitute a new level to need indentation. They should just be aligned, using spaces.

[spaces]: http://i.imgur.com/Nmw4Zr4.png
[tabs]: http://i.imgur.com/qa21IoX.png